webpack.config.js

```diff
@@ -28,6 +29,7 @@ module.exports = {
     path: path.resolve(__dirname, `dist/${target}`),
     library: "rosbag",
     libraryTarget: "umd",
+    globalObject: "typeof self !== 'undefined' ? self : this",
```

Comment:
Might be worth leaving a comment linking to https://github.com/webpack/webpack/issues/6525#issuecomment-417580843